torchgeo/datasets/etci2021.py

```diff
@@ -171,25 +171,56 @@ def _load_files(self, root: str, split: str) -> List[Dict[str, str]]:
         folders = [os.path.join(folder, "tiles") for folder in folders]
         for folder in folders:
             vvs = sorted(glob.glob(os.path.join(folder, "vv", "*.png")))
-            vhs = sorted(glob.glob(os.path.join(folder, "vh", "*.png")))
-            water_masks = sorted(
-                glob.glob(os.path.join(folder, "water_body_label", "*.png"))
-            )
+            vhs = [
+                vv.replace("/vv/", "/vh/").replace("_vv.png", "_vh.png") for vv in vvs
+            ]
+            water_masks = [
+                vv.replace("/vv/", "/water_body_label/").replace("_vv.png", ".png")
+                for vv in vvs
+            ]
 
             if split != "test":
-                flood_masks = sorted(
-                    glob.glob(os.path.join(folder, "flood_label", "*.png"))
-                )
+                flood_masks = [
+                    vv.replace("/vv/", "/flood_label/").replace("_vv.png", ".png")
+                    for vv in vvs
+                ]
```

Comment:
Forward slashes seems to be causing issues in Windows.

Reply:
Yeah, thank you. I add the fix.
(there is no "vv" in region name for ETCI 2021)